setup.py

```diff
@@ -28,11 +28,11 @@
     keywords='django rest auth registration rest-framework django-registration api',
     zip_safe=False,
     install_requires=[
-        'Django>=3.2',
+        'Django>=3.2,<5.0',
```

Comment:
why? I use dj-rest-auth/allauth with django5 with no problems

Reply:
@iMerica Why was this change included to require Django<5.0? dj-rest-auth v5.0.2 worked well on Django v5.0